src/plots/cartesian/set_convert.js

```diff
@@ -417,6 +417,10 @@ module.exports = function setConvert(ax) {
     ax._min = [];
     ax._max = [];
 
+    // copy ref to fullLayout.separators so that
+    // methods in Axes can use it w/o having to pass fullLayout
+    ax.separators = fullLayout.separators;
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1431/commits/6e25640adeac74d6cc7ec9847a5a74a8daff40a9